packages/contracts/scripts/set-scalar.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+RPC_URL=${RPC_URL:-http://localhost:8545}
+OVM_GAS_ORACLE=0x420000000000000000000000000000000000000F
+
+SCALAR=${SCALAR:-1000000}
```

Comment:
Does it make sense to make this a required parameter, that way someone doesn't accidentally run it with the default?

Reply:
Good call, I've updated the script